### PR TITLE
Add GAPIC config for LRO

### PR DESCRIPTION
**Pull request by bjwatson**

### CODE REVIEW

**Comment by bjwatson** [comment]
@garrettjonesgoogle I have re-purposed this PR to just add the GAPIC config for LRO, so we can build GAPIC wrappers for it to assist the google-cloud teams.

File: gapic/api/artman_lro.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+common:
+  api_name: google-longrunning-operations
```
Comment (garrettjonesgoogle) [inline gapic/api/artman_lro.yaml]: I think I would prefer "google-longrunning". It matches the directory it's declared in.

Comment (bjwatson) [inline gapic/api/artman_lro.yaml]: Fair enough. That also shortens it without resorting to an arbitrary abbreviation (e.g. `lro`).

**Comment by bjwatson** [comment]
PTAL

**Comment by garrettjonesgoogle** [comment]
LGTM

**Pull request merged**